customizations/utils.py

```diff
@@ -159,9 +159,11 @@ def upload_file_dropbox(path_file, file_name_saved):
     return public_url
 
 
-def create_webhook(token):
+def create_webhook(token, url):
+    if 'localhost' in url:
+        url = 'http://custom-ticket-local.herokuapp.com'
```

Comment:
I think the url should be an environment variable

Reply:
we'll use a different tool to assign a valid URL to localhost.